### DISCUSSION TITLE
PiGPIOFactory stops my servo from rotating

### DISCUSSION THREAD

**Discussion by hongfeij** [gpiozero/gpiozero#1141] [Q&A]
Operating system: Raspberry Pi OS(64-bits)
Python version: 3.11.2
Pi model: Pi 4 Model B
GPIO Zero version: 2.0.1
Pin factory used: e.g. PiGPIO
Servo: SG90

I used PiGPIOFactory as the warning indicates, but it stops my servo from rotating, the servo works well with RPI.GPIO except jitter. I tried both remote and local factories, they are not working. Anyone knows why?

PWMSoftwareFallback: To reduce servo jitter, use the pigpio pin factory.See https://gpiozero.readthedocs.io/en/stable/api_output.html#servo for more info 

```python
from gpiozero import AngularServo
from gpiozero.pins.pigpio import PiGPIOFactory
from time import sleep

factory = PiGPIOFactory(host="172.26.21.48")
servo = AngularServo(27, min_angle=-90, max_angle=90, pin_factory=factory)

try:
    while True:
        servo.angle = -90
        sleep(1)
        servo.angle = 90
        sleep(1)
except KeyboardInterrupt:
    print("Program exited by user")
```

**Comment by BeanBooy** [comment]
Hey, facing the same problem here.
Any solution found?
Thank you in advance
